sklearn/model_selection/tests/test_search.py

```diff
@@ -1695,7 +1695,9 @@ def _run_search(self, evaluate):
     for attr in dir(gscv):
         if attr[0].islower() and attr[-1:] == '_' and \
            attr not in {'cv_results_', 'best_estimator_',
-                        'refit_time_'}:
+                        'refit_time_',
+                        'classes_'  # TODO: remove in 0.24
```

Comment:
why would this be needed? This seems really weird...

Reply:
because the loop above goes `for attr in dir(gscv):` and the search have a `classes_` property that delegate to the `best_estimator_`

Reply:
(and obviously because in this specific test, the best_estimator_ is a tree)